src/domain/chains/useCases/isTestChain.js

```diff
@@ -0,0 +1,16 @@
+import { BLOCKCHAINS } from '../../../constants';
+
+export default function isTestChain (chain) {
+  if (chain) {
+    const chainCode = typeof chain === 'string' ? chain : chain.code;
+    const isChainValid = Object.keys(BLOCKCHAINS).find(chainObj => chainObj === chainCode);
+
+    if (!isChainValid) {
+      return null;
+    }
+
+    return chainCode === BLOCKCHAINS.mocknet.code || chainCode === BLOCKCHAINS.regtest.code;
```

Comment:
Are bitcoin testnet and ethereum rompsten not considered test chains in this context? Or would this be better renamed as `isMockChain`?

Reply:
@AnthonyRonning hmm I'll need your guidance here as I'm not super aware of which is test or mocknet.

Reply:
Regtest is similar to mocknet in that it's not on a live mainnet or testnet chain. 

https://bitcoin.org/en/developer-examples#regtest-mode

regtest is essentially a private bitcoin network, so there's no remote hash compare or anything like that, so verification would act like mocknet. 

I read the callers and it seems that `isMockChain` would be better if we want to also consider regtest a type of mock chain as well. `isTestChain` seems like it's talking about testnet chains and that's not true in this case. 

I think if we agree on that, then `NETWORK.testnet` should be changed to `NETWORK.mocknet` as well, since it's only considering `mocknet` and `regtest`.